django/contrib/gis/geos/polygon.py

```diff
@@ -86,7 +86,7 @@ def _create_polygon(self, length, items):
         n_holes = length - 1
         if n_holes:
             holes = (GEOM_PTR * n_holes)(*[self._clone(r) for r in rings])
-            holes_param = byref(holes)
```

Comment:
Is calling `c_uint()` redundant as well?
https://github.com/django/django/blob/b0d16d0129b7cc5978a8d55d2331a34cb369e6c7/django/contrib/gis/geos/polygon.py#L93